.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+    "env": {
+        "browser": true,
```

Comment:
questo a cosa serve ?

Reply:
ehm non so, la configurazione di eslint l'ha fatta automaticamente

Reply:
ok credo non serva, cerca un po'

Reply:
effettivamente sembra funzionare anche senza

Reply:
serve a iniettare nella runtime di jest le variabili globali presenti nel browser (che non esistono in JavaScript puro).